lib/engines/content_object_store/test/unit/app/models/content_block_edition_test.rb

```diff
@@ -10,14 +10,14 @@ class ContentObjectStore::ContentBlockEditionTest < ActiveSupport::TestCase
     @details = { "some_field" => "some_content" }
     @title = "Document title"
 
-    @content_block_edition = build(
-      :content_block_edition,
-      :email_address,
+    @content_block_edition = ContentObjectStore::ContentBlockEdition.new(
```

Comment:
We switch from using the factory here as we want to test that ActiveRecord sets up the Edition and Document objects correctly, rather than FactoryBot.

Reply:
Nice 👍

Reply:
I actually only found it because our stub of `create_random_id` was no longer being called 😬